irc.py

```diff
@@ -257,13 +260,52 @@ def command(self, cmd: bytes) -> None:
             print('Unknown command: ', cmd)
 
 
+def ip2long(ip):
+    """
+   Convert an IP string to long
+   """
+    packedIP = socket.inet_aton(ip)
+    return struct.unpack("!L", packedIP)[0]
+
 
 def main():
+    ###############################
+    # t == token-filename
+    # i == IP-Address
+    # p == port-number
+    # u == no-userlist
+    ###############################
+    utoken=''
+    uip='127.0.0.1'
+    uport='9007'
+    uuser=''
+ 
+    # Read command line args
+    myopts, args = getopt.getopt(sys.argv[1:],"t:i:p:u:")
```

Comment:
Can you have a look at the `argparse` module for reading command line? 

It's more high level. getopt is more or less like in C. That one can have help strings in the parameters already, enforce types, set default values.